### PR TITLE
Landing page doc

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the issue #29

### CODE REVIEW

File: docs/src/user/manual/about.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+[![Home Page](./img/about-img-1.png)](./img/about-img-1.png)
+
+1. **About:** Click on the `About` tab from the navigation bar to access the about page.
+
+## The Where?
+
+South Africa’s unique freshwater biodiversity is under enormous pressure from human activities, habitat transformation, climate change and invasive species impacts. River health is deteriorating faster than it can be measured, and existing data suggest that human impacts have, and continue to, severely compromise biodiversity and aquatic ecosystem function. This can also have serious adverse consequences for ecosystem services, such as the provision of food and safe, clean drinking water.
+
+## The what?
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/about.md:27]:
```suggestion
## The What?
```

File: docs/src/user/manual/help.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    [![ Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=9JI2D1c-1Jk)
+
+*  **Preparing and adding water temperature time series data:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=9JI2D1c-1Jk)
+
+* **Preparing an occurrence data upload file:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=hWoFoPBdWI4)
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/help.md:82]: Please embed the youtube videos using the mkdocs-videos plugin.

Comment (ketankartoza) [reply docs/src/user/manual/help.md:82]: I have done it in my previous PR 
This branch is made from the user manual branch that's why it is showing here as well.
I will rebase this branch from the main, resolve conflicts, and prepare this PR for review.

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Pull request merged**